freqtrade/data/converter.py

```diff
@@ -13,34 +13,31 @@
 logger = logging.getLogger(__name__)
 
 
-def parse_ticker_dataframe(ticker: list, timeframe: str, pair: str, *,
-                           fill_missing: bool = True,
-                           drop_incomplete: bool = True) -> DataFrame:
+def ohlcv_to_dataframe(ohlcv: list, timeframe: str, pair: str, *,
+                       fill_missing: bool = True, drop_incomplete: bool = True) -> DataFrame:
```

Comment:
this namingchange gives me headaches.

on one hand, it shouldn't matter as it's a internal function ...
on the other hand it's useful to test something "real quick" ... and i suspect i'm not the only one importing and using this directly.

Reply:
how do you use it? (in the interactive environment, I assume...)

Reply:
if it's interactive or a snippet of code i copy/paste into a REPL is not really relevant, is it?

Reply:
I mean, I do not use it along, you use it with some functions before and some functions after.
So the call to this function can be added into more convenient ones, to be called interactively or from a snippet, leaving this one internal.

> this namingchange gives me headaches.

But why? What's wrong with the new name?

Reply:
it's the change itself - not the old or new name ... but as you'll have seen i did approve so it's good to go (however i don't have time today to fix the merge conflict ...).